bee-block/src/protocol.rs

```diff
@@ -30,10 +31,16 @@ pub struct ProtocolParameters {
     token_supply: u64,
 }
 
+impl Borrow<()> for ProtocolParameters {
```

Comment:
Guess that's also needed for packable? A comment would be good, otherwise it looks like it should be deleted :trollface:

Reply:
Yes, will add a comment 👍🏻